connect4/riddles.ts

```diff
@@ -0,0 +1,130 @@
+import {Riddle, RiddlesLevel} from '../common/common';
```

Comment:
The levels are good but since it only takes one move I think the scale should be easier.
Code seems good!